app/views/indicators/index.html.erb

```diff
@@ -94,41 +77,55 @@
                    other_class: '-no-wrap',
                    column_key: 'definition' %>
       </div>
+      <div class="small-2 columns">
+        <%= render 'shared/components/c_order_filter_button',
+                   text: 'Type',
+                   other_class: '-no-wrap',
+                   column_key: 'type' %>
+      </div>
     </div>
   </div>
   <% @indicators.each do |indicator| %>
     <div class="c-table-list-item">
       <div class="row">
         <div class="small-2 columns">
-          <div class="c-table-list-item__text f-ff1-m-bold"><%= link_to indicator.alias, model_indicator_url(@model, indicator) %></div>
-        </div>
-        <div class="small-2 columns">
-          <div class="c-table-list-item__text f-ff1-m"><%= indicator.category %></div>
+          <div class="c-table-list-item__text f-ff1-m-bold"><%= indicator.name %></div>
```

Comment:
I think this should be `indicator.alias`, which is the long name with pipes (same applies to the parent name). I realise it's different in the design, but I have a feeling that the "name" part is meaningless without the category & subcategory parts (in any case it is not unique) . -any thoughts @juancarlosalonso - this is about the indicator name and "based on" system indicator name in the indicators list?